Dashboards first 20 lines optimization (#4084)
elastic/docs-content: explore-analyze/dashboards/drilldowns.md

# Add drilldowns [drilldowns]

> Panels have built-in interactive capabilities that apply filters to the dashboard data. For example, when you drag a time range or click a pie slice, a filter for the time range or pie slice is applied. **Drilldowns** let you customize and extend this interactive behavior by defining what happens when users click on data points, all while keeping the context of the interaction such as filters, time ranges, and selected values.

Reviewer: ```suggestion
Panels have built-in interactive capabilities that apply filters to the dashboard data. For example, when you drag a time range slider or click a pie chart slice, this applies a filter for the time range or pie slice. **Drilldowns** let you customize and extend this interactive behavior by defining what happens when you click on data points, while keeping interaction context such as filters, time ranges, and selected values.
```

## Requirements [drilldowns-requirements]

To add drilldowns to dashboard panels, you need:

> * **Edit** permissions for the **Dashboard** feature in {{product.kibana}}

Author: ```suggestion
* **All** privilege for the **Dashboard** feature in {{product.kibana}}
```

> * **Edit** permissions for the **Dashboard** feature in {{product.kibana}}

Reviewer: ```suggestion
* **Edit** permissions for the **Dashboards** feature in {{product.kibana}}
```
